grafana/resource_data_source.go

```diff
@@ -125,6 +125,11 @@ source selected (via the 'type' argument).
 							Optional:    true,
 							Description: "(Stackdriver) The authentication type: `jwt` or `gce`.",
 						},
+						"azure_log_analytics_same_as": {
+							Type:        schema.TypeBool,
+							Optional:    true,
+							Description: "(Azure Monitor) The log analytics same as.",
+						},
```

Comment:
I just saw that this one is deprecated now. https://github.com/grafana/grafana/blob/23956557d8c6a119b7de5be5c42024e29634d002/pkg/tsdb/azuremonitor/loganalytics/azure-log-analytics-datasource.go#L150-L153. We can leave it out

Reply:
Sorry :pray:  This field is removed